lib/danger/danger_core/plugins/dangerfile_github_plugin.rb

```diff
@@ -92,6 +92,14 @@ def self.instance_name
       "github"
     end
 
+    # @!group PR Review
+    # Current PR Review object.
+    # @return [GitHubReview]
+    #
```

Comment:
Sure, do you think we should provide an example here or what are your thoughts? 😄